### DISCUSSION TITLE
Custom time format

### DISCUSSION THREAD

**Discussion by Euro20179** [ppy/osu#17763] [Ideas] [answered]
I like having a custom time format.

I've basically already written the code for this functionality, but `CONTRIBUTING` says I should open a discussion.

First question is, should this be added?

My thoughts:
Currently the way I have it is that, in the language section of the settings menu (right below the 24 hour time option), there's an input box where you can input the time.
I have other ideas for how this could work. Currently it's a bit of an advanced feature, so maybe have an advanced section in the settings. 
Another option would be to make the input simpler, eg surrounding the part you want formatted using % signs, instead of just letting c# handle it. The problem with this is that, it's still kind of advanced.

I have the code [here](https://github.com/euro20179/osu/tree/prefer-time-format) if you wish to see.

**Comment by peppy** [comment, marked as answer]
No. This is a rhythm game. Use one of the provided time formats or fork the project and adjust to your liking, please.
